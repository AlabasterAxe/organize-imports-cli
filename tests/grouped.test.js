const runCli = require("./run-cli");

describe("grouped imports", () => {
  runCli("cli/grouped", ["file.js"]).test();
});
