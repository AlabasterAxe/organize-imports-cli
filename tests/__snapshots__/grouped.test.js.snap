// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grouped imports (status) 1`] = `0`;

exports[`grouped imports (stderr) 1`] = `""`;

exports[`grouped imports (stdout) 1`] = `
"Organizing imports...
<dir>/file.js/*CR*/
<dir>/file.js (modified)
Done!
"
`;

exports[`grouped imports (write) 1`] = `
Array [
  Object {
    "content": "// fs
import fs from \\"fs\\";
// other
import path from \\"path\\";
import util from \\"util\\";

console.log(path, fs, util);
",
    "filename": "file.js",
  },
]
`;
